Doc/library/contextlib.rst

```diff
@@ -137,6 +137,23 @@ Functions and classes provided:
    ``page.close()`` will be called when the :keyword:`with` block is exited.
 
 
+.. function:: nullcontext(thing=None)
```

Comment:
Minor: I’m nonplussed by the param name `thing`, I can’t think of a precedent in the stdlib.

Reply:
`obj` is probably more common.

Reply:
For this, I'd ask that we make the name explicit and call it `enter_result`.